### ISSUE TITLE
triplet loss formula

### ISSUE THREAD

**Issue by mangushev** [Walleclipse/Deep_Speaker-speaker_recognition_system#9]
Hi,
In triplet_loss, why it is 
loss = K.maximum(san - sap + alpha, 0.0) 
it got to be sap - san + alpha
Am I missing something?
Thanks!

**Comment by Walleclipse** [comment]
In general, our goal is to get higher anchor-positive similarity (sap) and lower anchor-negative similarity (san) . Thus loss function is opposite to object function, loss = K.maximum(san - sap + alpha, 0.0), that means, loss is high if I have higher  san and lower sap.
Check details in the **part 3.3** of [deep speaker paper](https://arxiv.org/pdf/1705.02304.pdf)

**Comment by mangushev** [comment]
Thanks, I was looking into euclidean similarity formula!
